packages/crawlers/src/index.ts

```diff
@@ -1,4 +1,34 @@
 export * from '@crawlers/core';
+// This export doesn't use * because of type collisions with /core, yet re-exporting manually works
```

Comment:
i would just `@ts-ignore` this if we want to reexport everything, this sounds error prone in such case

but what are the conflicts? maybe we should not export something from the core package? we dont really need to export things like `keys`, `entries`, `Constructor` or `Dictionary`, those are purely internal (and if something, they can be exported from one of those packages, not from both)

Reply:
That's a good point, I'll fix the core exports instead 👍